sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -40,7 +40,7 @@ import (
 var ecrRegex = regexp.MustCompile(`(^[a-zA-Z0-9][a-zA-Z0-9-_]*)\.dkr\.ecr\.([a-zA-Z0-9][a-zA-Z0-9-_]*)\.amazonaws\.com(\.cn)?.*`)
 
 func init() {
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
I have tested this change on host-ctr [PR](https://github.com/bottlerocket-os/bottlerocket/pull/3757)